pinot-spi/src/main/java/org/apache/pinot/spi/trace/Tracer.java

```diff
@@ -43,6 +43,14 @@ public interface Tracer {
      */
     InvocationScope createScope(Class<?> clazz);
 
+    /**
+     * Starts
+     * @return the request record
+     */
+    default RequestScope createRequestScope() {
```

Comment:
Within our internal code that receives the query rest request and calls broker request handle code, we do the following

```
RequestStatistics requestStatistics = new RequestStatistics();
brokerRequestHandler.handleRequest(request, requesterIdentity, requestStatistics);
```

I am guessing the first will be changed to following and it should continue to work as today ?

`Tracing.getTracer().createRequestScope()`

Reply:
Yes, you can do that, or just construct `new DefaultRequestStatistics()`, which is the old class which has moved to `pinot-spi`, if you don't want to use the `Tracer` SPI though this would break if you do use it and change the implementation. 

I'm surprised this is currently used as an API, so it's probably a good thing that this is being turned into an SPI.